module-7-web-apis/todo/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "todo",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "watch": "webpack --watch",
+    "start": "webpack-dev-server --open",
+    "build": "webpack"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "clean-webpack-plugin": "^2.0.2",
+    "css-loader": "^2.1.1",
+    "file-loader": "^3.0.1",
+    "html-webpack-plugin": "^3.2.0",
+    "light-server": "^2.6.2",
+    "lite-server": "^2.4.0",
```

Comment:
Are lite-server and light-server used somewhere?

Reply:
Good catch. I used those before webpack to be able to start the app on a web server. Now it can be deleted.